lib/Dialect/HW/HWOps.cpp

```diff
@@ -603,22 +604,21 @@ void HWModuleOp::build(OpBuilder &builder, OperationState &odsState,
                        HWModuleBuilder modBuilder, ArrayAttr parameters,
                        ArrayRef<NamedAttribute> attributes,
                        StringAttr comment) {
-  build(builder, odsState, name, ports, parameters, attributes, comment);
+  build(builder, odsState, name, ports, parameters, attributes, comment,
+        /*shouldEnsureTerminator=*/false);
   auto *bodyRegion = odsState.regions[0].get();
   OpBuilder::InsertionGuard guard(builder);
   builder.setInsertionPointToStart(&bodyRegion->front());
```

Comment:
Not introduced in this change, but why does this set the insertion point, only to set it again right away?
(AFAICT `builder` isn't used and no IR is changed in the constructor of the accessor)

Consider setting it just once.

Reply:
Good point!